tools/editor/io_plugins/editor_scene_import_plugin.cpp

```diff
@@ -656,6 +658,9 @@ void EditorSceneImportDialog::_choose_file(const String& p_path) {
 
 
 	import_path->set_text(p_path);
+	if (root_node_name->get_text().size()==0){
+		root_node_name->set_text(p_path.get_file().basename());
```

Comment:
Could you also prefill the LineEdit with the filename, using LineEdit's placeholder text feature?